wonnx/templates/pool/reduce.wgsl

```diff
@@ -40,7 +40,9 @@ fn main(@builtin(global_invocation_id) global_id: vec3<u32>) {
 		Now for each reduced axis, iterate all values and reduce. Note, starting value may not always be zero. For 
 		ReduceMin/Max we should initialize as NaN and keep a flag to check if we have seen at least one element -#}
 
-		var accumulator = {% if op_type == "ReduceProd" %} {{ scalar_type }}(1) {% else %} Scalar() {% endif %}; 
+		var accumulator = {% if op_type == "ReduceProd" %} {{ scalar_type }}(1) {% else %} Scalar() {% endif %};
+	    var max_element: Scalar = log(Scalar());
```

Comment:
Hm, perhaps add a note explaining why you initialize to `log(Scalar())` (I assume it is a trick to initialize to `-Inf` so the first value is always higher).

Reply:
Good point, will do!